CoreLayersView/src/au/gov/asd/tac/constellation/views/layers/shortcut/LayersViewShortcuts.java

```diff
@@ -0,0 +1,121 @@
+    @ActionReference(path = "Shortcuts", name = "CA-9")
+})
+@NbBundle.Messages("CTL_LayersViewShortcuts=Layers View: Shortcuts")
+public class LayersViewShortcuts extends AbstractAction {
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        Future<?> enableLayerFuture = null;
+
+        final String hotkey = e.getActionCommand();
+        switch (hotkey) {
+            case "CA-L":
+                PluginExecution.withPlugin(new NewLayerPlugin()).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-D":
+                PluginExecution.withPlugin(new DeselectAllLayersPlugin()).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-1":
+                enableLayerFuture = PluginExecution.withPlugin(new EnableLayerPlugin(1)).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-2":
+                enableLayerFuture = PluginExecution.withPlugin(new EnableLayerPlugin(2)).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-3":
+                enableLayerFuture = PluginExecution.withPlugin(new EnableLayerPlugin(3)).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-4":
+                enableLayerFuture = PluginExecution.withPlugin(new EnableLayerPlugin(4)).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-5":
+                enableLayerFuture = PluginExecution.withPlugin(new EnableLayerPlugin(5)).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-6":
+                enableLayerFuture = PluginExecution.withPlugin(new EnableLayerPlugin(6)).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-7":
+                enableLayerFuture = PluginExecution.withPlugin(new EnableLayerPlugin(7)).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-8":
+                enableLayerFuture = PluginExecution.withPlugin(new EnableLayerPlugin(8)).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            case "CA-9":
+                enableLayerFuture = PluginExecution.withPlugin(new EnableLayerPlugin(9)).executeLater(GraphManager.getDefault().getActiveGraph());
+                break;
+            default:
+                break;
+        }
+
+        // wait for future, execute and update the state.
+        if (enableLayerFuture != null) {
+            try {
+                enableLayerFuture.get();
+            } catch (InterruptedException | ExecutionException ex) {
+                Exceptions.printStackTrace(ex);
+                // Restore interrupted state...
+                Thread.currentThread().interrupt();
```

Comment:
Split the `InterruptedException` and `ExecutionException` cases as there is no need to interrupt the thread in the `ExecutionException` case